### DISCUSSION TITLE
Support for Amazfit GTR3 / GTR3PRO / GTS3

### DISCUSSION THREAD

**Discussion by Klaus3d3** [NightscoutFoundation/xDrip#1980] [Ideas]
Has anyone started with the developement of a watchface app for these watches?

Since they have a new OS called ZeppOS a lot of things seems to be possible.

I'm registered as an individual developer at HUAMIs developer program and have access to the documentation, some examples and Zepp Studio.

We now have BLECommunication with the companion app, clickable items on the watchface, eventlisteners and so on. Looks really promissing to hook on, at least, onto xDrips webservice easily and get a longlasting, functional smartwatch.

But honestly i need some help with this Javascript shit.

**Comment by parapenT1sta** [comment]
Try to leave a comment here: https://bigdigital.home.blog/2022/01/27/a-new-supported-device-amazfit-gts2-mini/
Maybe [bigdigital](https://bigdigitalhome.wordpress.com/) can help you.
